### PR TITLE
Random password generator

### PR DESCRIPTION
**Pull request by Namyalg**
Script to generate a password given length

### CODE REVIEW

File: Random_password_generator/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Random Password Generator
+- This script can be used to generate a strong randomly password
+- The length of the password is passed as a Command Line argument
+
```
Comment (yashaswibiyahut) [inline Random_password_generator/README.md:4]: Please add an example/sample use case for how to use this script.

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]
@Namyalg  add a running rode.
and also add proper comments in your code.

File: Random_password_generator/README.md (hunk 1 above)
Comment (Namyalg) [reply Random_password_generator/README.md:4]: I have made the changes, please let me know if there is anything else I have to change

**Comment by yashaswibiyahut** [comment]
@Namyalg Kindly add a running code

**Comment by Namyalg** [comment]
What exactly does running code mean?

**Comment by yashaswibiyahut** [comment]
@Namyalg there is error in your code.

**Comment by Namyalg** [comment]
It works fine for me

**Comment by yashaswibiyahut** [comment]
IMG_20201023_200320

**Comment by Namyalg** [comment]
It has to be passed as a command line argument,  i will change it to user
input then

Namya LG

**Comment by Namyalg** [comment]
@yashaswibiyahut , can I create a PR again, I had to delete my repository and fork it again

**Comment by tejan-singh** [comment]
close this PR and create another one.

**Pull request closed without merging**